pre_commit_hooks/terraform/validate.sh

```diff
@@ -7,6 +7,9 @@ trap 'exit' ERR
 # Make environment variables working in OSX GUI apps such as Github Desktop https://stackoverflow.com/q/135688/483528
 export PATH=$PATH:/usr/local/bin
 
+# set default aws region for validating aws providers
+export AWS_REGION=${AWS_REGION:="none"}
```

Comment:
cool, we could also append to the terraform validate command instead of exporting to the environment. But that decision, I leave to you! :)

Reply:
more readable this way i guess... ;) thx